src/app/demos/data-entry-grid/data-entry-grid-demo.component.ts

```diff
@@ -107,11 +107,14 @@ export class SkyDataEntryGridDemoComponent implements OnInit {
   public gridApi: GridApi;
   public gridOptions: GridOptions;
   public searchText: string;
+  public noRowsTemplate: string;
 
   constructor(
     private agGridService: SkyAgGridService,
     private modalService: SkyModalService
-  ) { }
+  ) {
+    this.noRowsTemplate = `No results found`;
```

Comment:
```suggestion
    this.noRowsTemplate = `No results found.`;
```
This is a complete sentence, so I'd add a period to the end. ... I assume this is text to display in the grid when someone enters search criteria that doesn't have any matches. Is that correct?

Reply:
fixed in commit [1056b86 - "add period"](https://github.com/blackbaud/skyux2-docs/pull/1076/commits/1056b8667fd4f31b26eb2143579fbfdb3bd71198)